src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,72 @@
+package duke.command;
+
+import java.time.format.DateTimeParseException;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+
+/**
+ * Represent commands Todo, Deadline, Event which add tasks to the list.
+ *
+ * @author Armando Jovan Kusuma
+ */
+public class AddCommand extends Command {
+    private String command;
+
+
+    public AddCommand(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Executes the command.
+     *
+     * @param tasks The TaskList where the command is to be executed.
+     * @param ui The UI which functions as the user interface of the Chat bot.
+     * @param storage The storage file to store the list of tasks.
+     * @throws DukeException If the command execution fails.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        if (command.equalsIgnoreCase("todo")) {
```

Comment:
As a suggestion, may you could extract the lower-case command into a separate variable and use `switch` statements to handle the different cases instead.